### PR TITLE
Add IAM samples to pubsub.

### PR DESCRIPTION
**Pull request by jeffmendoza**

### CODE REVIEW

**Comment by jeffmendoza** [comment]
@remi  PTAL

**Comment by jeffmendoza** [comment]
@bshaffer I think remi is ooo, can you take a look?

File: pubsub/sample.rb [hunk 1]
```diff
@@ -106,3 +106,89 @@ def list_subscriptions
   end
 end
 # [END list_subscriptions]
+
+# [START print_topic_policy]
+def print_topic_policy
```
Comment (remi) [inline pubsub/sample.rb]: Rename method and doc region tag to `get_topic_policy`

File: pubsub/sample.rb [hunk 2]
```diff
@@ -106,3 +106,89 @@ def list_subscriptions
...
+
+  policy = topic.policy
+
+  puts "Topic policy:"
+  puts policy.roles
+end
+# [END print_topic_policy]
+
+# [START print_subscription_policy]
+def print_subscription_policy
```
Comment (remi) [inline pubsub/sample.rb]: Rename method and doc region tag to `get_subscription_policy`

**Comment by remi** [comment]
2 naming nits for consistency.  Otherwise LGTM

File: pubsub/sample.rb [hunk 3]
```diff
@@ -106,3 +106,89 @@ def list_subscriptions
   end
 end
 # [END list_subscriptions]
+
+# [START print_topic_policy]
```
Comment (remi) [inline pubsub/sample.rb]: Please put doc region tags inside methods so the code sample shows the relevant code (not wrapped in a method)

Comment (jeffmendoza) [inline pubsub/sample.rb]: All the existing tags are around the methods https://cloud.google.com/pubsub/publisher

Do you want me to change all of them?

Comment (remi) [inline pubsub/sample.rb]: Ah, you're right!  I forgot these were live 😄 

Yeah, let's go ahead and change them.  It looks really nice without the method definition.  Eg. the way Python samples show up on that page

Comment (jeffmendoza) [inline pubsub/sample.rb]: Sounds good, will do.

**Comment by jeffmendoza** [comment]
@remi Comments addressed, ptal

**Pull request merged**